### PR TITLE
Import api

### PR DESCRIPTION
**Pull request by Brolyus**
Few changes: planets form uses the API, forms have a default value inviting the user to select a parameter, and this default value is disabled once clicked. Cleaned the code a bit, now next step will be to use "unstructured props"

### CODE REVIEW

File: astra/src/components/DwarfPlanetData.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      "Hauméa est une planète naine située dans la ceinture de Kuiper. Elle a une forme ovale allant de 1960 à 2500 km de diamètre. Une année dure 285 ans sur Hauméa. Elle possède 2 satellites : Hi'iaka et Namka. Distance avec le Soleil : 6,5 milliards de km.",
+    image: "https://www.slsc.org/wp-content/uploads/2018/09/Haumea.H11.1452x816-jgt-960x540.png"
+  },
+];
+
+const DwarfPlanetData = () => (
+  <div className='Lexique' >
+   <Lexique/>
+    {DwarfPlanets.map(item => (
+      <Lexicon name={item.name} description={item.description} image={item.image} />
```
Comment (ghost) [inline astra/src/components/DwarfPlanetData.js]: Missing unique key in list

File: astra/src/components/Game.js [hunk 2]
```diff
@@ -69,23 +49,50 @@ class Game extends React.Component {
...
+            options.englishName === "Deimos" ||
+            options.englishName === "Io"
+          ) {
+            this.setState({
+              planetList: [...this.state.planetList, options]
+            });
+          }
+        })
+      )
+      .then(this.setAllPlanets);
```
Comment (ghost) [inline astra/src/components/Game.js:54]: this.setAllPlanets not used ?

File: astra/src/components/Game.js [hunk 3]
```diff
@@ -69,23 +49,50 @@ class Game extends React.Component {
...
+  }
+
+  setAllPlanets() {
+    this.setState({
+      planetOptions: this.state.planetList.map(option => option.name.replace())
```
Comment (ghost) [inline astra/src/components/Game.js]: .replace() : what's the point ?

File: astra/src/components/Game.js [hunk 4]
```diff
@@ -152,13 +176,34 @@ class Game extends React.Component {
...
-          La distance entre {this.state.depart} et {this.state.arrival} est de{" "}
-          {this.state.distance} km
-        </p>
-        <p>
-          En {this.state.vehicle}, il faut {this.state.time} heures pour faire
-          le trajet, soit {Math.floor(this.state.time / 24)} jours
+        <br />
+        <br />
+        <br />
+        <br />
```
Comment (ghost) [inline astra/src/components/Game.js]: No way to handle this in CSS ?

File: astra/src/components/Game.js [hunk 5]
```diff
@@ -69,23 +49,50 @@ class Game extends React.Component {
...
     this.handleSubmitNewVehicle = this.handleSubmitNewVehicle.bind(this);
+    this.setAllPlanets = this.setAllPlanets.bind(this);
+  }
+  componentDidMount() {
+    fetch("https://api.le-systeme-solaire.net/rest/bodies")
+      .then(response => response.json())
+      .then(data =>
+        data.bodies.filter(options => {
+          if (
+            options.isPlanet === true ||
```
Comment (ghost) [inline astra/src/components/Game.js]: You do not need to write === true to verify if condition is true

File: astra/src/components/DwarfPlanetData.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Lexicon from "./Lexicon";
+import Lexique from "./Lexique";
+import './Lexique.css';
+
+
+const DwarfPlanets = [
```
Comment (ghost) [inline astra/src/components/DwarfPlanetData.js]: Move those data in a separate data file

File: astra/src/components/Game.js [hunk 7]
```diff
@@ -2,55 +2,34 @@ import React from "react";
...
-  {
-    id: "mercure",
-    name: "Mercure",
-    englishName: "Mercury",
-    distance: 57909227
-  }
-];
+import "./Game.css";
 
 const vehicleList = [
```
Comment (ghost) [inline astra/src/components/Game.js]: Move those data in a separate data file

File: astra/src/components/GameFormVehicle.css [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+  text-shadow: 1px 1px 3px black;
+
+}
+
+.Vehicle, select{
+  display: flex;
+  align-content: space-around;
+}
+
+/* .LabelVehicle{
```
Comment (ghost) [inline astra/src/components/GameFormVehicle.css]: dead code

File: astra/src/components/Lexique.js [hunk 9]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+//import './Lexique.css';
```
Comment (ghost) [inline astra/src/components/Lexique.js]: dead code ?

File: astra/src/components/Navbar.css [hunk 10]
```diff
@@ -1,13 +1,37 @@
...
+  text-align: left;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+
+/* .navbarList{
```
Comment (ghost) [inline astra/src/components/Navbar.css]: Dead code is dead

File: astra/src/components/SatelliteData.js [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  {
+    name: "",
+    description:
+      "",
+    image: ""
+  },
+];
+
+const SatelliteData = () => (
```
Comment (ghost) [inline astra/src/components/SatelliteData.js:7]: Separate data and the Component in two different files

File: astra/src/components/SunData.js [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+    image: "https://vignette.wikia.nocookie.net/thesolarsystem6361/images/5/59/Sun_spacepedia.png/revision/latest?cb=20180301152819" 
+  },
+];
+
+const SunData = () => (
```
Comment (ghost) [inline astra/src/components/SunData.js:7]: In two different files, separate Component and data

**Review by ghost — CHANGES_REQUESTED** [review]

File: astra/src/components/Game.js (hunk 3 above)
Comment (Brolyus) [reply astra/src/components/Game.js]: Corrected, .replace is part of a next branch, it's now deleted: 

> setAllPlanets() {
    this.setState({
      planetOptions: this.state.planetList.map(option => option.name)
    });
  }

File: astra/src/components/Game.js (hunk 5 above)
Comment (Brolyus) [reply astra/src/components/Game.js]: Deleted the === true, now it looks like: 

> `          if (
            options.isPlanet ||
            options.englishName === "Moon" ||
            options.englishName === "Sun" ||
            options.englishName === "Deimos" ||
            options.englishName === "Io"
          ) {`

File: astra/src/components/DwarfPlanetData.js (hunk 1 above)
Comment (Brolyus) [reply astra/src/components/DwarfPlanetData.js]: Corrected, now .map also add a key: 

`{DwarfPlanets.map(item => (
      <Lexicon key= {item.name} name={item.name} description={item.description} image={item.image} />`

File: astra/src/components/Game.js (hunk 4 above)
Comment (Brolyus) [reply astra/src/components/Game.js]: Deleted, only 1 <br /> remaining, it was already handled by css

File: astra/src/components/Game.js (hunk 7 above)
Comment (Brolyus) [reply astra/src/components/Game.js]: planets data are now from API call, so this array is deleted.
Vehicles are now stored in data/vehicleList.js

File: astra/src/components/GameFormVehicle.css (hunk 8 above)
Comment (Brolyus) [reply astra/src/components/GameFormVehicle.css]: deleted the dead code

File: astra/src/components/DwarfPlanetData.js (hunk 6 above)
Comment (Brolyus) [reply astra/src/components/DwarfPlanetData.js]: created a dwarfPlanets.js file in a data folded, stored the datas in it, now DwarfPlanetData.js is cleanest

File: astra/src/components/Lexique.js (hunk 9 above)
Comment (Brolyus) [reply astra/src/components/Lexique.js]: deleted dead code

File: astra/src/components/Navbar.css (hunk 10 above)
Comment (Brolyus) [reply astra/src/components/Navbar.css]: deleted this dead code.

File: astra/src/components/SatelliteData.js (hunk 11 above)
Comment (Brolyus) [reply astra/src/components/SatelliteData.js:7]: created satellite.js in data folder, and stored this datas in it

File: astra/src/components/SunData.js (hunk 12 above)
Comment (Brolyus) [reply astra/src/components/SunData.js:7]: stored datas in sun.js in data folder

**Comment by Brolyus** [comment]
Pushed a corrected version of this branch

**Review by ghost — APPROVED** [review]

**Pull request merged**